mailpoet/assets/js/src/settings/pages/key_activation/key_activation.tsx

```diff
@@ -19,6 +19,70 @@ type KeyState = {
   is_approved: boolean;
 }
 
+function Messages(
+  state:KeyActivationState,
+  showPendingApprovalNotice:boolean,
+  activationCallback:()=>Promise<void>
```

Comment:
Apart from the already mentioned `argument: type` spacing, the `=>` arrow should have one space on each side. Interesting that our eslint doesn't catch that. Anyway, Prettier will solve it.